build/CMakeLists.txt

```diff
@@ -355,6 +373,15 @@ if(${TRTIS_ENABLE_METRICS})
   set(TRTIS_DEPENDS ${TRTIS_DEPENDS} prometheus-cpp)
 endif() # TRTIS_ENABLE_METRICS
 
+if(${TRTIS_ENABLE_S3})
```

Comment:
Put this at line 365 just after GCS (and follow style of no blank lines before/after)